src/main/java/at/porscheinformatik/sonarqube/licensecheck/LicenseCheckSensor.java

```diff
@@ -49,6 +51,8 @@ public LicenseCheckSensor(
                 ),
                 new MavenDependencyScanner(licenseMappingService),
                 new GradleDependencyScanner(licenseMappingService),
+                new GradleDependencyScanner(licenseMappingService),
```

Comment:
You have duplicated `GradleDependencyScanner`

Reply:
Duplicate entry removed. Thanks!